composer.json

```diff
@@ -18,7 +22,8 @@
         "onelogin/php-saml": "^3.0.0"
     },
     "require-dev": {
-        "mockery/mockery": "0.9.*"
+        "mockery/mockery": "0.9.*",
+        "phpunit/phpunit": "~4.0"
```

Comment:
this can be `<8` which will resolve to 7 for php 7, and 6 for 5.6 I believe. -  https://github.com/aacotroneo/laravel-saml2/compare/aa-test?expand=1